go/tricorder/metric_test.go

```diff
@@ -1737,6 +1775,16 @@ func TestListNoChangeSubType(t *testing.T) {
 	t.Error("panic expected")
 }
 
+func TestNoAssignedUnitNoAdd(t *testing.T) {
+	bucketer := NewGeometricBucketer(1, 1000)
+	dist := newDistribution(bucketer, false)
+	defer func() {
+		recover()
```

Comment:
My first inclination is to test the result of recover is equal to some string, but since the docs don't specify the exact panic message, won't that make my tests to fragile? I could just test that the panic is a string, but all my panics are strings. Thoughts?